presto-cassandra/src/main/java/com/facebook/presto/cassandra/util/SslContextProvider.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cassandra.util;
+
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManagerFactory;
+
+import java.io.FileInputStream;
+
+import java.security.KeyStore;
+import java.security.SecureRandom;
+
+public class SslContextProvider
+{
+    public SSLContext getSSLContext(String truststorePath,
+                                    String truststorePassword,
+                                    String keystorePath,
+                                    String keystorePassword)
+            throws Exception
+    {
+        FileInputStream tsf = new FileInputStream(truststorePath);
```

Comment:
in the Presto codebase, we don't use abbreviations -- please use full words (e.g. trustStoreInputStream)